exporter/prometheus.go

```diff
@@ -46,3 +66,55 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 	log.Info("All Metrics successfully collected")
 
 }
+
+func (e *Exporter) isTokenExpired() (bool, error) {
+	u := *e.APIURL()
+	u.Path = path.Join(u.Path, "rate_limit")
+
+	resp, err := getHTTPResponse(u.String(), e.APIToken())
+
+	if err != nil {
+		return false, err
+	}
+	defer resp.Body.Close()
+	// Triggers if rate-limiting isn't enabled on private Github Enterprise installations
+	if resp.StatusCode == 404 {
+		return false, errors.New("404 Error")
+	}
+
+	limit, err := strconv.ParseFloat(resp.Header.Get("X-RateLimit-Limit"), 64)
+
+	if err != nil {
+		return false, err
+	}
+
+	defaultLimit := os.Getenv("GITHUB_RATE_LIMIT")
+	if len(defaultLimit) == 0 {
+		defaultLimit = "15000"
+	}
```

Comment:
there's a helper function in the `go-common` library we already use here that can do this for you:
```suggestion
	defaultLimit := cfg.GetEnv("GITHUB_RATE_LIMIT", "15000")
```